src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -249,7 +248,7 @@
     <Compile Include="ParsersAndFormatters\TestData.cs" />
     <Compile Include="ParsersAndFormatters\TestUtils.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsNetCoreApp)' == 'true'">
+  <ItemGroup>
```

Comment:
Having this ItemGroup with 1 element and without condition looks bad specially as all the other ItemGroups have no conditions as well.